pom.xml

```diff
@@ -65,7 +65,7 @@
         <dependency>
             <groupId>net.smartcosmos</groupId>
             <artifactId>smartcosmos-user-entity-devkit</artifactId>
-            <version>3.0.1-SNAPSHOT</version>
+            <version>3.0.0</version>
```

Comment:
It should be: https://github.com/SMARTRACTECHNOLOGY/smartcosmos-user-entity-devkit/blob/master/pom.xml#L17